distribution/docs/src/main/resources/_contents/_configuring/configuring-global-settings-contents.adoc

```diff
@@ -103,30 +103,48 @@ Does not change the address the system runs on.
 |HTTPS Port
 |org.codice.ddf.system.httpsPort
 |String
-|The https port used by the system.
+|The https port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8993
 |Yes
 
 |HTTP Port
 |org.codice.ddf.system.httpPort
 |String
-|The http port used by the system.
+|The http port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8181
 |Yes
 
 |Default Port
 |org.codice.ddf.system.port
 |String
-|The default port used to advertise the system. This should match either the http or https port.
+|The default port used to advertise the system. This should match either the above http or https port.
 
 NOTE: Does not change the port the system runs on.
 |8993
 |Yes
 
+|Internal HTTPS Port
```

Comment:
I was actually just looking to see whether we had anything in the installation guide that talked about those types of scenarios. I couldn't find anything related to it though. It might be a good to talk about running behind a proxy or load balancer in the install guide somewhere